server/models/schema.sql

```diff
@@ -612,24 +612,21 @@ CREATE TABLE `employee` (
   `email`         VARCHAR(70) DEFAULT NULL,
   `fonction_id`   TINYINT(3) UNSIGNED DEFAULT NULL,
   `service_id`    SMALLINT(5) UNSIGNED DEFAULT NULL,
-  `location_id`   BINARY(16) NOT NULL,
   `creditor_uuid` BINARY(16) DEFAULT NULL,
-  `debtor_uuid`   BINARY(16) DEFAULT NULL,
   `locked`        TINYINT(1) DEFAULT NULL,
+  `patient_uuid`  BINARY(16) DEFAULT NULL,
   PRIMARY KEY (`id`),
   UNIQUE KEY `employee_1` (`code`),
   KEY `fonction_id` (`fonction_id`),
   KEY `service_id` (`service_id`),
-  KEY `location_id` (`location_id`),
   KEY `creditor_uuid` (`creditor_uuid`),
-  KEY `debtor_uuid` (`debtor_uuid`),
```

Comment:
Okay - the `patient_uuid` will still link to a `debtor`, so employees are still both debtors/creditors in BHIMA after this change.  I was just thinking that getting the `debtor_uuid` for an employee will now require that we do an extra JOIN on `patient`.